rust/cubestore/cubestore/Cargo.toml

```diff
@@ -34,10 +34,10 @@ bytes = "0.5.4"
 serde_json = "1.0.56"
 futures = "0.3.5"
 smallvec = "1.8.0"
-flexbuffers = "0.1.1"
```

Comment:
Do we need to upgrade flexbuffers as part of this PR?

Reply:
as mentioned in general comments, build fails everywhere unless I do this. Are there concerns about this upgrade?